processing-service-core/global-config-options.md

```diff
@@ -18,13 +20,14 @@ The body of the request would then contain the strings *default* and *descriptio
 
 	{
 	  "default": "string",
-	  "description": "string"
+	  "description": "string",
+	  "scope": "TENANT"
 	}
 
 A request issued to create or update a global configuration can return:
 
 - **204** status code - when successful and returns no content,
-- **400** status code - when invalid key is used.
+- **400** status code - when invalid key is used or some of required properties is missing in the body of the request.
 
 ## Retrieving global configuration
 
```

Comment:
"when successful and returns the string value of the global configuration" - that's not correct. It's an object.